package.json

```diff
@@ -201,7 +201,7 @@
 		"node-watch": "0.6.4",
 		"postcss-color-function": "4.1.0",
 		"postcss-loader": "3.0.0",
-		"prettier": "npm:wp-prettier@2.0.5",
+		"prettier": "npm:wp-prettier@2.0.5-beta-1",
```

Comment:
Is this not a downgrade?

Reply:
@psealock 🤦 you're right.

Also, I noticed your fix still fails on CI for the error, so maybe our CI env is a little different. 😧

Reply:
So my confusion around beta-1 being an update was that its the latest version tag by over a month in that repo. But no idea if this is a mistake in the tag or something else.